CHANGELOG.md

```diff
@@ -20,6 +37,10 @@
 
 [Full Changelog](https://github.com/theforeman/puppet-dhcp/compare/5.0.0...5.0.1)
 
+**Breaking changes:**
```

Comment:
I think this is a concurrency bug in the changelog generator and is really part of this release. If needed, use the git master version which has fix it.